### PR TITLE
Making static previews of website content

### PR DESCRIPTION
**Pull request by abitrolly**
For static previews we need to start website in background, wait for it to finish and scrape it with something. Waiting for web site could be done in several ways:

1. sleep 5  - hacky
2. use `supervisord`
3. use `pexpect`

It is also possible to use django testing harness to start site somehow.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -2,7 +2,6 @@
 elasticsearch==2.1.0
 Django==1.9.8
 djangorestframework==3.4.0
-psycopg2==2.6.2
```
Comment (davojta) [inline requirements/base.txt:5]: Why you delete psycopg2 from base requirements? will postgre still  be working on local machine ?

Comment (abitrolly) [reply requirements/base.txt:5]: dev settings uses sqlite inherited from https://github.com/kinaklub/next.filmfest.by/blob/80b6c23d8c84d78ec7b6b7b263f7ddc84d326a63/filmfest/settings/base.py#L96 so local configuration doesn't use it.

File: requirements/pytest.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # consciously added packages
--r base.txt
+-r prod.txt
```
Comment (staaas) [inline requirements/pytest.txt]: i hesitate about this change

Comment (abitrolly) [reply requirements/pytest.txt]: Well, I didn't want to insert pg bindings into every file. The reasoning is that requirements is not configuration, so production will not be affected if its dependencies are used by tests.

**Comment by staaas** [comment]
@abitrolly is this still WIP? or should we review this?

**Comment by abitrolly** [comment]
@nott not ready yet - I still have to figure out how to wait for web site to appear properly.

**Comment by abitrolly** [comment]
32 commits for 55 changed lines. I gonna squash it to new PR for review.

**Pull request closed without merging**